index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+	<title>Home | Chandler Morrison</title>
+	<link href="https://fonts.googleapis.com/css?family=Inconsolata|Montserrat|Oxygen" rel="stylesheet">
+<ul class="NavBar">
+	<li><a href="index.html">Home</a></li>
+	<li><a href="bio.html">Bio</a></li>
+
+</ul>
+
+
+
+
+
+
+
+
+
+
+
+
+</head>
+
+<body background="images/Jacket.jpg">
+<h1>Chandler Morrison</h1>
+<h1>Me at NSS</h1>
+<p>Hi, my name is Chandler Morrison. I'm currently a student at Nashville Software School.</p>
+
+<!-- Blog posts -->
+<h3>1st week</h3>
+<section>The first week was filled with nervous energy and excitement. I'm so happy to be finally learning something that i'm interested in, not to mention working with computers. Our primary instructor, Joe, was out for the week and was temporarily replaced by Steve Brownlee. He will show up on Tuesday.</section>
+
+<h3>2nd week</h3>
+<section>The second week was a shorter week due to memorial day, we worked more on html/css as we finished our first group project. The point of the projects were to give us a feel of what it feels like to work in a group enviroment and to get used to working in a group in github. I quickly learned that everyday would cause my brain to melt. But that's ok because at least I enjoyed learning what I was learning.</section>
+
+<h3>3rd week</h3>
+<section>The third week continued to melt my brain with the start of javascript. We were talking about so many concepts so quickly it was hard to keep up. But it was nice to hear from Joe that it was ok that I'm confused and that I didn't need to know all of what we were talking about. Because it felt like all the info I was learning went in one ear and out the other.</section>
+
+
+<footer id="AdjustedFooter">
+  Chandler Morrison &copy; 2017
+
+<a href="https://twitter.com/cptr95">
+<img class="twitter" src="images/Twitter.png" alt="" >
+
+<a href="https://www.facebook.com/chandler.morrison.33">
+<img src="images/facebook.png" alt="Facebook">
+
+  </footer>
+  <script type="main.js"></script>
```

Comment:
It doesn't look like javascript is being used to populate the DOM. Make sure you read through the requirements and fulfill each of them.